src/dotnet-compile-fsc/AssemblyInfoOptionsCommandLine.fs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Microsoft.DotNet.Tools.Compiler
```

Comment:
Who's going to implement this msbuild task? Are they doing it or are we going to?

Is there going to be a `dotnet cli` extensibility tool built on top of this msbuild task?